nltk/test/binary_fixt.py

```diff
@@ -0,0 +1,11 @@
+from nltk.internals import find_binary
+
+
+def check_binary(binary):
+    import pytest
+
+    try:
+        find_binary(binary)
+    except:
+        pytest.skip("Skipping test because the {binary} binary was not found")
+        return False
```

Comment:
We don't need to `return False` here, right? It would make you think that the succeeding case returns `True`, but doing so would require the doctests to be:
```python
>>> check_binary(...)
True
>>> [actual tests]
...
```

Reply:
Ok, removed False.